src/fides/api/models/privacy_request.py

```diff
@@ -294,6 +299,7 @@ class PrivacyRequest(
     cancel_reason = Column(String(200))
     canceled_at = Column(DateTime(timezone=True), nullable=True)
     consent_preferences = Column(MutableList.as_mutable(JSONB), nullable=True)
+    source = Column(String, nullable=True)
```

Comment:
Can we instead use a `Column(EnumColumn(PrivacyRequestSource), nullable=True)` here? This way we can still have the benefit of explicitly defining what we should expect in this column without having the enums formalized in the DB? 

See how we do something similar with `method = Column(EnumColumn(ConsentMethod))` in our `PrivacyPreferenceHistory` model